barretenberg/cpp/src/barretenberg/smt_verification/terms/ffiterm.cpp

```diff
@@ -38,56 +38,52 @@ FFITerm::FFITerm(const std::string& t, Solver* slv, bool isconst, uint32_t base)
         slv->s.assertFormula(ge);
         slv->s.assertFormula(lt);
     } else {
-        std::string tmp = slv->s.mkFiniteFieldElem(t, slv->fp, base).getFiniteFieldValue(); // dumb but works
-        if (tmp[0] == '-') {
-            this->term = slv->s.mkTerm(cvc5::Kind::ADD, { slv->s.mkInteger(tmp), this->modulus });
-        } else {
-            this->term = slv->s.mkInteger(tmp);
-        }
-        // this->term = slv->s.mkInteger(tmp); won't work for now since the assertion will definitely fail
+        std::string strvalue = slv->s.mkFiniteFieldElem(t, slv->fp, base).getFiniteFieldValue(); // TODO(alex): works for now
```

Comment:
Could you add a more descriptive TODO? What do you wan to do here in the future?